tutor/src/models/tour/region.js

```diff
@@ -15,7 +14,7 @@ export default class TourRegion extends BaseModel {
   @field({ type: 'array' }) otherTours;
 
   @computed get tour_ids() {
-    return concat( [this.id], this.otherTours.peek() );
+    return this.otherTours.reverse().concat( [this.id] ).reverse();
   }
```

Comment:
if you change this back, and then push onto `otherTours`, `tour_ids` never reacts, while this does.  weirdly, `[this.id].concat(this.otherTours)` does not work either.  I mean, maybe we dont really need `tour_ids` to be reactive...i'd added this cause i thought i might want to use this as a mechanism for the post-publish wheels, but then i did.

but i figured since `otherTours` is an observable, that the `computed`s based off that should be too.